### DISCUSSION TITLE
Best effort traffic query

### DISCUSSION THREAD

**Discussion by ananyaalways** [CoRE-RG/CoRE4INET#37] [Q&A] [answered]
Hello,
Do you have the feature that if for example, the gates of 4 queues are open at the same time, the highest priority will get the data through and only when the higher priority queue does not have traffic,  the lower priority queues get a chance to transmit data.

**Comment by Musteblume** [comment, marked as answer]
Yes. The transmission selection always chooses the not empty queue with the highest priority.
